### PR TITLE
Correct output states validation

### PR DESCRIPTION
**Pull request by gregdingle**
# Description

Some of the needed states were missing from validation. 

# Tests

sql query against prod database for all actual values

### CODE REVIEW

File: app/models/output_state.rb [hunk 1]
```diff
@@ -16,11 +16,28 @@ class OutputState < ApplicationRecord
...
+    PipelineRun::STATUS_CHECKED,
+    PipelineRun::STATUS_FAILED,
     PipelineRun::STATUS_LOADED,
-    PipelineRun::STATUS_UNKNOWN,
     PipelineRun::STATUS_LOADING,
-    PipelineRun::STATUS_LOADING_QUEUED,
     PipelineRun::STATUS_LOADING_ERROR,
+    PipelineRun::STATUS_LOADING_QUEUED,
+    PipelineRun::STATUS_RUNNING,
+    PipelineRun::STATUS_UNKNOWN,
```
Comment (MarkAZhang) [inline app/models/output_state.rb]: Hey Greg, I couldn't find any evidence that STATUS_RUNNING and STATUS_CHECKED are valid output states.

Comment from pipeline_run.rb:
The output_states indicate the state of each target output, the progression being as follows:
UNKNOWN -> LOADING_QUEUED -> LOADING -> LOADED / FAILED 

(the docs are missing LOADING_ERROR)

Comment (MarkAZhang) [reply app/models/output_state.rb]: I think the only one that needs to be added is STATUS_FAILED.

Comment (gregdingle) [reply app/models/output_state.rb]: > I couldn't find any evidence that STATUS_RUNNING and STATUS_CHECKED are valid output states

The states I guess are legacy. They are only found older than 2019. Also `NULL`. 

Good investigation!

**Comment by gregdingle** [comment]
@MarkAZhang ready for review again.

**Pull request merged**